packages/core/src/workflows/step.ts

```diff
@@ -17,6 +18,7 @@ export type ExecuteFunctionParams<TStepInput, TResumeSchema, TSuspendSchema, Eng
   inputData: TStepInput;
   resumeData?: TResumeSchema;
   runCount: number;
+  aiTracingContext: AITracingContext;
```

Comment:
can we name this `tracingContext`. I think it makes more sense since this will be the default way of doing things

Reply:
sure, but I'd prefer to do in a different PR since this is used outside this PR too.